<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stm32bootloaderDialog</class>
 <widget class="QDialog" name="stm32bootloaderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="hexFilePathApplyBtn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QLabel" name="adressLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Adress:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="adressLineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>311</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="hexFilePathLabel">
      <property name="text">
       <string>.hex file:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="hexFilePathLineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="hexFilePathSelectBtn">
      <property name="maximumSize">
       <size>
        <width>31</width>
        <height>21</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="downloadUrlPushbutton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>from url</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
